### PR TITLE
Env suggestions

### PR DESCRIPTION
**Pull request by JohnAsaro**
suggested changes to .env and config.py and merging the json_dictonary branch into develop

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+AIRTABLE_SERVICE = "tbl2hG3F6LlSW3Bm7"
+AIRTABLE_BIKESforSALE = "tblWAo0tdOEFN2RaA"
+AIRTABLE_BIKESforDONO = "tblGlO7tDFuQg3RKy"
+AIRTABLE_RENTALBIKES = "tbllDlYHfpxvWi5XH"
+AIRTABE_WEBSHOPINQUIRY = "tblaKhFw5X6cvBNNp"
+AIRTABLE_SALESORDERS = "tblp14q73VINEXGM1"
+AIRTABLE_ORDERITEMS = "tblCKoA3zEjflGoLf"
+AIRTABLE_INTERACTIONS = "tblYDL11X6PuvlnKA"
+AIRTABLE_NOLONGERUSERSALES = "tbldLNHp78lSBKaxZ"
+AIRTABLE_RENTAL = "tblATiIu8nRsN9JSW"
```
Comment (awoolfson) [inline .env.example:15]: I liked the idea you had of adding these to the config instead, I think that would be better. We should leave the TEST keyword in, and just add an underscore BETWEEN_EACH_WORD_LIKE_THIS. The reason is that it is more readable that way and we are accessing a test database after all. Once we start actually deploying and they use the program on their actual database we can create new constants for the real deal.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+urllib3==2.0.4
+Werkzeug==2.3.7
+zope.component==6.0
+zope.deferredimport==5.0
+zope.deprecation==5.0
+zope.event==5.0
+zope.hookable==5.4
+zope.interface==6.0
+zope.lifecycleevent==5.0
+zope.proxy==5.0.0
```
Comment (awoolfson) [inline requirements.txt:35]: Do we need all of these? We only have a few dependencies so far

File: src/config.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+AIRTABLE_SERVICE = (os.getenv("AIRTABLE_SERVICE")) #-> str: #initalizes service table
+AIRTABLE_BIKESforSALE = (os.getenv("AIRTABLE_BIKESforSALE")) #-> str: #initalizes initializes bikes for sale table
+AIRTABLE_BIKESforDONO = (os.getenv("AIRTABLE_BIKESforDONO")) #-> str: #initalizes bikes for donation table
+AIRTABLE_RENTALBIKES = (os.getenv("AIRTABLE_RENTALBIKES")) #-> str: #initalizes rental bikes table
+AIRTABE_WEBSHOPINQUIRY = (os.getenv("AIRTABE_WEBSHOPINQUIRY")) #-> str: #initalizes web shop inquiry table
+AIRTABLE_SALESORDERS = (os.getenv("AIRTABLE_SALESORDERS")) #-> str: #initalizes sales orders table
+AIRTABLE_ORDERITEMS = (os.getenv("AIRTABLE_ORDERITEMS")) #-> str: #initalizes order items table
+AIRTABLE_INTERACTIONS = (os.getenv("AIRTABLE_INTERACTIONS")) #-> str: #initalizes interactions table
+AIRTABLE_NOLONGERUSERSALES = (os.getenv("AIRTABLE_NOLONGERUSERSALES")) #-> str: #initalizes no longer user sales table
+AIRTABLE_RENTAL = (os.getenv("AIRTABLE_RENTAL")) #-> str: #initalizes rental table
```
Comment (awoolfson) [inline src/config.py:19]: Add hard-coded strings instead

**Review by awoolfson — COMMENTED** [review]
Please leave the naming conventions the way I put them, I find it a bit more clear and easier to read. Your suggestion to move the table and base ids out of env was good, those can be put in config

File: requirements.txt (hunk 2 above)
Comment (JohnAsaro) [reply requirements.txt:35]: Not entirely sure, I just did pip freeze, we most likely only need dot-env, flask, requests,pprintpp and npm. This is just to be safe.

**Comment by JohnAsaro** [comment]
> Please leave the naming conventions the way I put them, I find it a bit more clear and easier to read. Your suggestion to move the table and base ids out of env was good, those can be put in config

gotcha, this good on the variable names?

File: .env.example (hunk 1 above)
Comment (JohnAsaro) [reply .env.example]: oh my bad I thought that was the real database

**Review by awoolfson — APPROVED** [review]
I'm going to pull this and make a few changes, thanks

**Pull request closed without merging**